packages/core/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@aries-framework/core",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.0.0",
+  "files": [
+    "build"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://github.com/hyperledger/aries-framework-javascript/tree/main/packages/core",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperledger/aries-framework-javascript",
+    "directory": "packages/core"
+  },
+  "scripts": {
+    "build": "yarn run clean && yarn run compile",
+    "clean": "rimraf -rf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "yarn run build"
+  },
+  "dependencies": {
+    "abort-controller": "^3.0.0",
+    "bn.js": "^5.2.0",
+    "borc": "^3.0.0",
+    "buffer": "^6.0.3",
+    "class-transformer": "^0.4.0",
+    "class-validator": "^0.13.1",
+    "js-sha256": "^0.9.0",
+    "make-error": "^1.3.6",
+    "multibase": "^4.0.4",
+    "multihashes": "^4.0.2",
+    "object-inspect": "^1.10.3",
+    "reflect-metadata": "^0.1.13",
+    "rxjs": "^7.1.0",
+    "tsyringe": "^4.5.0",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "@types/bn.js": "^5.1.0",
+    "@types/events": "^3.0.0",
+    "@types/indy-sdk": "^1.16.5",
+    "@types/node-fetch": "^2.5.10",
```

Comment:
I guess we will eventually remove this from the core, right?

Reply:
Yes once we resolve the typing issues as described above (declare them ourselves or use some trick to make it work)